tools/shared.py

```diff
@@ -6,7 +6,7 @@
 from __future__ import print_function
 
 from distutils.spawn import find_executable
-from subprocess import PIPE, STDOUT
+from subprocess import PIPE, STDOUT # noqa
```

Comment:
why is this needed?

Reply:
There are no more direct uses of STDOUT in this file.. so this is an unused import.. but other modules expect to be able to import shared.STDOUT.